test/Microsoft.Dnx.Tooling.FunctionalTests/DnuPackTests.cs

```diff
@@ -6,575 +6,596 @@
-                    Assert.Equal(0, exitCode);
-                    Assert.True(Directory.Exists(Path.Combine(workingDir, "bin")));
-                    Assert.True(File.Exists(Path.Combine(workingDir, "bin", "Debug", "dnx451", "fr-FR", "ReadFromResources.resources.dll")));
-                    Assert.True(File.Exists(Path.Combine(workingDir, "bin", "Debug", "dnxcore50", "fr-FR", "ReadFromResources.resources.dll")));
-                }
-            }
+            Assert.Empty(result.StandardError);
+            Assert.Contains(expectedDNX, result.StandardOutput);
+            Assert.Contains(expectedDNXCore, result.StandardOutput);
+            Assert.Contains(expectedNupkg, result.StandardOutput);
+            Assert.True(Directory.Exists($"{projectPath}/bin"));
         }
 
-        [Theory]
-        [MemberData(nameof(RuntimeComponents))]
-        public void DnuPack_ResourcesNoArgs_WarningAsErrorsCompilationOption(string flavor, string os, string architecture)
-        {
-            string stdOut;
-            string stdError;
-            var runtimeHomeDir = TestUtils.GetRuntimeHomeDir(flavor, os, architecture);
-            int exitCode;
-
-            using (var testEnv = new DnuTestEnvironment(runtimeHomeDir))
-            {
-                using (var tempDir = new DisposableDir())
-                {
-                    var appPath = Path.Combine(tempDir, "ResourcesTestProjects", "ReadFromResources");
-                    TestUtils.CopyFolder(Path.Combine(TestUtils.GetMiscProjectsFolder(), "ResourcesTestProjects", "ReadFromResources"), appPath);
-                    var workingDir = Path.Combine(appPath, "src", "ResourcesLibrary");
-
-                    var environment = new Dictionary<string, string> { { "DNX_TRACE", "0" } };
-                    DnuTestUtils.ExecDnu(
-                        runtimeHomeDir,
-                        "restore", "",
-                        out stdOut,
-                        out stdError,
-                        environment: null,
-                        workingDir: workingDir);
-                    exitCode = DnuTestUtils.ExecDnu(
-                        runtimeHomeDir,
-                        "pack",
-                        "",
-                        out stdOut,
-                        out stdError,
-                        environment,
-                        workingDir);
-
-                    Assert.Empty(stdError);
-                    Assert.Equal(0, exitCode);
-                    Assert.True(Directory.Exists(Path.Combine(workingDir, "bin")));
-                    Assert.True(File.Exists(Path.Combine(workingDir, "bin", "Debug", "dnx451", "fr-FR", "ResourcesLibrary.resources.dll")));
-                    Assert.True(File.Exists(Path.Combine(workingDir, "bin", "Debug", "dnxcore50", "fr-FR", "ResourcesLibrary.resources.dll")));
-                }
-            }
-        }
+        //    [Theory]
+        //    [MemberData(nameof(RuntimeComponents))]
```

Comment:
See my comment about porting to the new framework. I'm rolling that back anyway and just porting the one new test. I figured that rewriting each file as we touch it would be a reasonable balance between a big-scale rewrite and only using the framework for new tests, but I'm just not sure I have the time.